pkg/cvo/cvo_test.go

```diff
@@ -3130,7 +3135,9 @@ func TestOperator_upgradeableSync(t *testing.T) {
 			optr.cvLister = &clientCVLister{client: optr.client}
 			optr.upgradeableChecks = optr.defaultUpgradeableChecks()
 
-			err := optr.upgradeableSync(optr.queueKey())
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
```

Comment:
If we aren't setting a timeout, I don't see why we can't just use:

```go
ctx := context.Background()
```

in these test functions.

Reply:
fixed